src/BenchmarksBot/Tags.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BenchmarksBot
+{
+    /// <summary>
+    /// Contains the mapping between scenario names and GitHub labels to assign to issues related to each scenario.
+    /// </summary>
+    public class Tags
+    {
+        public static readonly Dictionary<string, Tags> Scenarios = new Dictionary<string, Tags>(StringComparer.OrdinalIgnoreCase)
+        {
+            {"BasicViews.GetTagHelpers", new Tags { Labels = new [] { "area-mvc" } } },
+            {"BasicViews.Post", new Tags { Labels = new [] { "area-mvc" } } },
+            {"ConnectionClose", new Tags { Labels = new [] { "area-servers" } } },
+            {"DbFortunesDapper", new Tags { Labels = new [] { "" } } },
+            {"DbFortunesEf", new Tags { Labels = new [] { "" } } },
+            {"DbFortunesRaw", new Tags { Labels = new [] { "" } } },
+            {"DbMultiQueryDapper", new Tags { Labels = new [] { "" } } },
+            {"DbMultiQueryEf", new Tags { Labels = new [] { "" } } },
+            {"DbMultiQueryRaw", new Tags { Labels = new [] { "" } } },
+            {"DbMultiUpdateDapper", new Tags { Labels = new [] { "" } } },
+            {"DbMultiUpdateEf", new Tags { Labels = new [] { "" } } },
+            {"DbMultiUpdateRaw", new Tags { Labels = new [] { "" } } },
+            {"DbSingleQueryDapper", new Tags { Labels = new [] { "" } } },
+            {"DbSingleQueryEf", new Tags { Labels = new [] { "" } } },
+            {"DbSingleQueryRaw", new Tags { Labels = new [] { "" } } },
+            {"EndpointPlaintext", new Tags { Labels = new [] { "area-mvc" } } },
+            {"FortunesPostgreSql-Actix", new Tags { Labels = new [] { "" } } },
+            {"FortunesPostgreSql-FastHttp", new Tags { Labels = new [] { "" } } },
+            {"FortunesPostgreSql-NodeJs", new Tags { Labels = new [] { "" } } },
+            {"HttpClient", new Tags { Labels = new [] { "area-httpclientfactory" } } },
+            {"HttpClientFactory", new Tags { Labels = new [] { "area-httpclientfactory" } } },
+            {"HttpClientParallel", new Tags { Labels = new [] { "area-httpclientfactory" } } },
+            {"Json", new Tags { Labels = new [] { "" } } },
```

Comment:
We should probably assign *some* kind of label for all the issues, otherwise they'll end up in the routing queue and some human will have to route them anyway. Maybe we need an `area-perf` for other things and we cover that in the perf sync?

Reply:
Ok with adding `area-perf` as a default, though it wouldn't add more value than the new default label we are also adding with this pr (`perf`, `perf-xyz-reason`).

I will start with what is there, and now it will be easy to just change what is created.

I am also thinking of adding more properties to this metadata class, like which script name is used in the jenkins so we can generate direct links, or anything else we would like to see in each issue related to a scenario.

Reply:
The only value it would add is that the routing logic (MikLabel) literally checks for labels starting `area-`.